plugins/tiddlywiki/markdown/wrapper.js

```diff
@@ -36,6 +36,10 @@ var accumulatingTypes = {
 	"text": true,
 	"softbreak": true
 };
+// If rendering WikiText, we treat katex nodes as text.
+if (pluginOpts.renderWikiText) {
```

Comment:
It's a really tiny thing, but our coding style does not put whitespace between `if` and the following parenthesis.

Reply:
Oops! It looks like this file uses the wrong style. Should I just change touched lines or the whole file?

Reply:
Oops indeed! I think it would be best to make it consistent. Thank you!

Reply:
No prob, done.